export.py

```diff
@@ -1779,6 +1779,21 @@ def has_emissive_material(db):
     return False
 
 
+def export_for_bake(db):
+    export = True
+    node_names = []
+    for mat in db.material:
+        if mat and mat.node_tree:
+            nt = mat.node_tree
+            for n in nt.nodes:
+                node_names.append(n.name)
+    if node_names:
+        if 'PxrBakeTexture' not in node_names:
+            export = False
+
```

Comment:
K.  couple things here.  

1.  The node names can comepletely change, should check bl_idname of node if it is PxrBakeTexturePatternNode
2.  You can make this a bit more simple:


for mat in db.material:
      if mat and mat.node_tree:
           for n in mat.node_tree.nodes:
                 if n.bl_idname == "PxrBakeTexturePatternNode":
                        return True
return False